ui/packages/shared/components/src/Pill/index.tsx

```diff
@@ -15,28 +15,28 @@ import cx from 'classnames';
 
 const VARIANTS = {
   primary: {
-    color: 'text-gray-600',
-    bg: 'bg-indigo-100',
+    color: 'text-gray-600 dark:text-gray-100',
+    bg: 'bg-indigo-100 dark:bg-indigo-800',
   },
   success: {
-    color: 'text-green-800',
-    bg: 'bg-green-100',
+    color: 'text-green-800 dark:text-green-100',
+    bg: 'bg-green-100 dark:bg-green-800',
   },
   danger: {
-    color: 'text-red-800',
-    bg: 'bg-red-100',
+    color: 'text-red-800 dark:text-red-100',
+    bg: 'bg-red-100 dark:bg-red-800',
   },
   warning: {
-    color: 'text-amber-800',
```

Comment:
Not a biggy, but to keep the shades of the colour consistent between the dark and light modes, I usually map `800` to `200`. 
Like: 

`900` -> `100`
`800` -> `200` 
`700` -> `300`
`600` -> `400` 
`500` -> `500`.